src/core/Akka.Remote/Transport/DotNetty/TcpTransport.cs

```diff
@@ -209,17 +209,18 @@ public TcpTransport(ActorSystem system, Config config) : base(system, config)
                 var handler = (TcpClientHandler)associate.Pipeline.Last();
                 return await handler.StatusFuture.ConfigureAwait(false);
             }
+            catch (ConnectException c)
```

Comment:
Fixed the failing test - simple change, really. Issue just was that DotNetty v0.4.8 doesn't necessarily throw an `AggregateException` with a `ConnectException` anymore; we'll just get the `ConnectException` itself in the `catch` block. So, modified those methods to handle that case explicitly.